lib/checks/canAddLikeTermWithCoefficientNodes.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if the nodes are terms with coefficients that can be added together.
+// The nodes need to have the same base and exponent
+// e.g. 2x + 5x, 6x^2 + x^2, nthRoot(4,2) + nthRoot(4,2)
+function canAddLikeTermWithCoefficientNodes(node, termWithCoefficientSubclass) {
+  if (!Node.Type.isOperator(node) || node.op !== '+') {
+    return false;
+  }
+  const args = node.args;
+  if (!args.every(n => Node.TermWithCoefficient.isTermWithCoefficient(
+    n, termWithCoefficientSubclass.baseNodeFunc))) {
+    return false;
+  }
+  if (args.length === 1) {
```

Comment:
that is... really weird haha. Can you either put it on your Asana board (and know that you'll work on it soon) or open an issue so I can remember to look into it?